src/normalize.rs

```diff
@@ -176,6 +180,16 @@ impl<'a> Filter<'a> {
                     }
                 }
             }
+
+            if self.normalization >= PathDependencies {
```

Comment:
From skimming https://github.com/diesel-rs/diesel/commit/ca34951b6a4f9ada94d43ada6f1567cd9d34c3b9, I would request 2 changes:

- Only use path dependency replacement for dependencies not in a subdirectory of the tested crate's workspace. I would prefer to keep `$WORKSPACE/diesel/src` instead of `$DIESEL/src`.

- This needs to match #85 by hiding line numbers from other crates. Touching unrelated code in the other crate should not require regenerating the stderr files.